src/main/java/reposense/util/FileUtil.java

```diff
@@ -38,6 +39,49 @@
     private static final String GITHUB_API_DATE_FORMAT = "yyyy-MM-dd";
     private static final ByteBuffer buffer = ByteBuffer.allocate(1 << 11); // 2KB
 
+    /**
+     * Zips all the relative folders and relevant files of {@code fileTypes} contained in
+     * {@code sourceAndOutputPath} directory into the same folder.
+     */
+    public static void zipRelativeFiles(HashSet<Path> relativePaths,
+                                        Path sourceAndOutputPath, String... fileTypes) {
+        zipRelativeFiles(relativePaths, sourceAndOutputPath, sourceAndOutputPath, fileTypes);
+    }
+
+    /**
+     * Zips all the relevant files and relative folders
+     * @param relativePaths contains the relevant folders to be zipped.
+     * @param fileTypes contains the file types to be zipped. Only files which are of the type "fileTypes" will be
+     *                  zipped.
+     */
+    public static void zipRelativeFiles(HashSet<Path> relativePaths,
+                                        Path sourcePath, Path outputPath, String... fileTypes) {
+        try (
+                FileOutputStream fos = new FileOutputStream(outputPath + File.separator + ZIP_FILE);
+                ZipOutputStream zos = new ZipOutputStream(fos)
```

Comment:
Weird style here too. Not sure if it is correct.

Reply:
Maybe our checkstyle is not checking this.

Reply:
Aparently the checkstyle isn't picking up those, seemingly how my local checkstyle and travis dosen't respond to these issues. I assume those checkstyle can be configured to pick up these other checkstyle issues as well?

Reply:
Yes, maybe you can raise an issue about this. Before that, just follow the code style in your newly added part and leave the rest there.

Reply:
I believe this current indentation is correct (8 spaces from the previous line since it's a parentheses and not curly brackets)? The same code in the `zip` method in the master branch shows the same code

https://github.com/reposense/RepoSense/blob/81db2aaf93d06bbf19be2efc2372baa6b5106957/src/main/java/reposense/util/FileUtil.java#L91-L97

Reply:
The code style in the main branch may not be correct, too. I have mentioned that checkstyle may not check this part. I will do some search tonight. Before that, just leave this as it.

Reply:
Ok noted

Reply:
This is a code use of try-with-resource and I do not find a code style from [oss-generic coding standards](https://oss-generic.github.io/process/docs/CodingStandards.html). That's what I found: [try with resources](https://stackoverflow.com/questions/30753996/what-is-the-code-convention-for-formatting-try-with-resources) basically. So, let the seniors decide which one is better.